app/views/epitaphs/share.html.erb

```diff
@@ -9,3 +9,67 @@
   <div><%= link_to t(:write_yours_from_share), root_path %>
 <% end %>
 
+<div id="threedeeplayland">
+    <script>
+    var SOONRIP = SOONRIP || {}
+    SOONRIP['ep'] = { name: "<%= @epitaph.name %>",
+                    epitaph:"<%= @epitaph.text %>"};
+    </script>
+
+    <div oncontextmenu="return false;" id="viewerContext" style="width: 50 vw; height: 50vh"></div>
```

Comment:
it's very handy css3.  i don't want it in the div markup anyways, but 1 plumbing problem at a time.